src/core/store/index.ts

```diff
@@ -281,6 +293,11 @@ namespace Store {
       selected?: string;
     }
   }
+
+  export interface Location {
```

Comment:
Perhaps rename to `Geolocation` to avoid ambiguity between browser `location` or some other location.